### DISCUSSION TITLE
Проблема с просмотром анонимных таблиц

### DISCUSSION THREAD

**Discussion by nshy** [totumonline/totum-ru-issues-and-discussions#547] [ОШБК!] [answered]
Привет!

Проблема продемонстрирована на скриншоте. В анонимной таблице около 500 записей. Отображается только часть, остальные не загружаются.

Наблюдаю на десктопе `chromium-124.0.6367.49-1`. На `андроиде-11` работает без проблем. Однако сообщали, что на некоторых андроидах та же проблема, версию не уточнял.

Версия Тотума `4.11.57.0` (последний на текущий момент). С `PRO` не пробовал.

![screenshot](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/1797510/0dc56366-5e3f-49d4-b187-9b94ba5714b9)

**Comment by ttm-alexeypolunin** [comment]
Да, подтверждаю, что не работает. Записал в список на исправление, но это будет в V5, который выйдет в конце июля это хороший вариант.

**Reply by ttm-alexeypolunin** [reply]
Ошибка связана именно с загрузкой строк в фронт — если поставить PRO Row buffering https://docs.totum.online/tables#table-settings-row_buffering то будет отображать.

**Reply by nshy** [reply]
Спасибо!

**Reply by ttm-alexeypolunin** [reply, marked as answer]
Исправлено V5
